tests/Main.idr

```diff
@@ -312,6 +312,7 @@ racketTests = MkTestPool "Racket backend" [] (Just Racket)
     , "semaphores001", "semaphores002"
     , "futures001"
     , "mutex001", "mutex002", "mutex003", "mutex004", "mutex005"
+    , "channels001", "channels002", "channels003", "channels004", "channels005", "channels006"
```

Comment:
Rather than duplicating the tests across the Chez and Racket test pools, I think we should group them under the `AllSchemes` test pool (assuming this doesn't break anything). Around `tests/Main.idr:187`.